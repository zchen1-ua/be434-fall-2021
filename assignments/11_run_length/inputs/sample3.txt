TTTGTAAAGTCTGGATTAACTGCTATAAAATCGGAAACCATAACACCTTTTAGAGTTAAA
GAATCTCCTGTTCAAATGGAATGTATTGTTAATGATGTTATTGAACTTGGAGACCAAGGT
GGAGCAGGAAATTTAGTAGTATGTGAAATAAAAATGATTCACATTAATGAAGATATTCTT
GATGATGAAGGAATTATTGATCCAAATAAAATTAAATTAGTCGGACGCATGGGTGGAAAC
TGGTATTGTAAAACTACCAACGAATCTATCTTTGAAGTTGTTAAACCTATCCGTAATTTA
GGTATTGGTGTTGATCAGATTCCTAAACGAATTAAAAATAGCTATATTCTTAGTGGAAAT
GATTTAGGTATGCTAGGAAATATAGAAGCCTTACCTACCATCGAAGAGGTTGAAGAATAC
AAAAAAGAAAACTAACACTATAAAATGAAATAATTAAATTTTAAGTAGTGAAGATGAAGA
CAATGCTGAATTTGAATAATTTCCATGTGAATTGGTGGAATATTGTTCTAACAGTGCACG
