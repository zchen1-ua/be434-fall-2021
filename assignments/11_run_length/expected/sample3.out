T3GTA3GTCTG2AT2A2CTGCTATA4TCG2A3C2ATA2CAC2T4AGAGT2A3
GA2TCTC2TGT2CA3TG2A2TGTAT2GT2A2TGATGT2AT2GA2CT2G2AGAC2A2G2T
G2AGCAG2A3T3AGTAGTATGTGA3TA5TGAT2CACAT2A2TGA2GATAT2CT2
GATGATGA2G2A2T2AT2GATC2A3TA4T2A3T2AGTCG2ACGCATG3TG2A3C
TG2TAT2GTA4CTAC2A2CGA2TCTATCT3GA2GT2GT2A3C2TATC2GTA2T3A
G2TAT2G2TGT2GATCAGAT2C2TA3CGA2T2A5TAGCTATAT2CT2AGTG2A3T
GAT3AG2TATGCTAG2A3TATAGA2GC2T2AC2TAC2ATCGA2GAG2T2GA2GA2TAC
A6GA4CTA2CACTATA4TGA3TA2T2A3T4A2GTAGTGA2GATGA2GA
CA2TGCTGA2T3GA2TA2T3C2ATGTGA2T2G2TG2A2TAT2GT2CTA2CAGTGCACG
